<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderParams</class>
 <widget class="QFrame" name="ShaderParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Node Layer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_nodeLayer"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_label"/>
   </item>
   <item>
    <widget class="QComboBox" name="m_visibility">
     <item>
      <property name="text">
       <string>external</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>internal</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
